Philips.CodeAnalysis.SecurityAnalyzers/LicenseAnalyzer.cs.backup

```diff
@@ -0,0 +1,100 @@
+﻿// © 2021 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using Microsoft.CodeAnalysis;

```

Comment:
We may need to support various types of license information, such as SPDX vs free-text. We can deal with that later.